From: =?utf-8?b?VGFhdmkgVsOkw6Ruw6RuZW4=?= <hi@taavi.wtf>
Date: Sat, 4 Jan 2025 13:47:27 +0200
Subject: Support Codeberg as a host

---
 make.go | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/make.go b/make.go
index 9f48c07..d43066b 100644
--- a/make.go
+++ b/make.go
@@ -161,6 +161,9 @@ func (u *upstream) tarballUrl() (string, error) {
 	case "git.sr.ht":
 		return fmt.Sprintf("%s/archive/%s.tar.%s",
 			repo, u.tag, u.compression), nil
+	case "codeberg.org":
+		return fmt.Sprintf("%s/archive/%s.tar.%s",
+			repo, u.tag, u.compression), nil
 	default:
 		return "", errUnsupportedHoster
 	}
@@ -579,6 +582,7 @@ func shortHostName(gopkg string, allowUnknownHoster bool) (host string, err erro
 		"blitiri.com.ar":       "blitiri",
 		"cloud.google.com":     "googlecloud",
 		"code.google.com":      "googlecode",
+		"codeberg.org":         "codeberg",
 		"filippo.io":           "filippo",
 		"fyne.io":              "fyne",
 		"git.sr.ht":            "sourcehut",
