From: Roger Shimizu <rosh@debian.org>
Date: Fri, 17 Jul 2020 22:55:35 +0900
Subject: Update the import path of golang-github-google-go-github

---
 main.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: dh-make-golang/main.go
===================================================================
--- dh-make-golang.orig/main.go	2022-11-18 12:45:52.035957432 +0100
+++ dh-make-golang/main.go	2022-11-18 12:45:52.035957432 +0100
@@ -4,7 +4,7 @@
 	"fmt"
 	"os"
 
-	"github.com/google/go-github/v38/github"
+	"github.com/google/go-github/github"
 	"github.com/gregjones/httpcache"
 )
 
Index: dh-make-golang/go.mod
===================================================================
--- dh-make-golang.orig/go.mod	2022-11-18 12:43:54.600226791 +0100
+++ dh-make-golang/go.mod	2022-11-18 12:46:09.780533980 +0100
@@ -4,7 +4,7 @@
 
 require (
 	github.com/charmbracelet/glamour v0.3.0
-	github.com/google/go-github/v38 v38.1.0
+	github.com/google/go-github v38.1.0
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79
 	github.com/mattn/go-isatty v0.0.12
 	golang.org/x/mod v0.5.1 // indirect
